modlog_wiki_publisher.py

```
@@ -1238,28 +1475,42 @@ def update_wiki_page(reddit, subreddit_name: str, wiki_page: str, content: str,
 
             # If new content would make page exceed limit, we need to handle it
             if existing_size > warning_threshold:
-                logger.warning(f"Existing wiki page already at {existing_size:,} bytes ({(existing_size/REDDIT_WIKI_LIMIT)*100:.1f}% of limit)")
+                logger.warning(
+                    f"Existing wiki page already at {existing_size:,} bytes ({(existing_size / REDDIT_WIKI_LIMIT) * 100:.1f}% of limit)"
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)